.circleci/config.yml

```diff
@@ -237,11 +237,6 @@ workflows:
               only:
                 - main
     jobs:
-      - hmpps/npm_outdated:
```

Comment:
removed this as it adds to alert fatigue and is not strictly necessary for repos that aren't the template project